evm-tests/jsontests/src/main.rs

```diff
@@ -188,11 +188,15 @@ const SKIPPED_CASES: &[&str] = &[
 	// custom json parser. https://github.com/ethereum/tests/issues/971
 	"stTransactionTest/ValueOverflow",
 	"stTransactionTest/ValueOverflowParis",
+	// It's impossible touch storage by precompiles
```

Comment:
With Aurora's custom precompiles, we might have some that touch storage. What is going wrong with these tests that fail?

Reply:
As tests itself is a black box, it means that I don't know the post-execution state which I should analyze. I got only `expected_hash`. And for that particular test, I can't get post-state anywhere. My reference state for analyzing is REVM tests. But it's skip it. @birchmd  If you know some useful tools, that can help rebuild the state after json-tests, it'll be really cool. As it's only one test to skip.
AFAIK in that tests **standard** precompiles somehow touch state. And it's impossible behavior. 
But if I'll know post execution state I can manage it.
More over, it affects only hard forks: only London or before London.

Reply:
If it doesn't impact the current hardforks then I suppose it's not an issue to skip these tests.